src/model/flatListItem.ts

```diff
@@ -0,0 +1,7 @@
+export type FlatListItem = {
```

Comment:
is this used anywhere? i can't see it?

Either way, for something like this it would be better to give it a name like `TrainTimesFlatListItem`, as it's likely that you'll have lots of different flatlists and the data in this one is train-time-specific, so the generic name it has at the moment could be quite misleading.

If it's not used, you can just delete it though